rct229/reports/ashrae9012019/ashrae901_2019_summary_report.py

```diff
@@ -12,24 +11,30 @@ def __init__(self):
         self.title = "ASHRAE STD 229P RULESET CHECKING TOOL"
         self.purpose = "Summary Report"
         self.ruleset = "ASHRAE 90.1-2019 Performance Rating Method (Appendix G)"
-        self.ruleset_report_file = "ashrae901_2019_summary_report.md"
+        self.ruleset_report_file = f"{self.__class__.__name__}.md"
 
     def initialize_ruleset_report(self, rule_outcome=None):
         self.section_list = [
             "All",
+            "Performance Calculations",
+            "Schedules Setpoints",
             "Envelope",
             "Lighting",
+            "HVAC General",
             "Receptacles",
             "Transformers",
             "HVAC-Baseline",
             "HVAC-General",
             "HVAC-HotWaterSide",
             "HVAC-ChilledWaterSide",
             "HVAC-AirSide",
-        ]
+        ]  # TODO: need to expand as more sections are developed
         self.section_dict = {
+            "1": "Performance Calculations",
+            "4": "Schedules Setpoints",
             "5": "Envelope",
             "6": "Lighting",
+            "10": "HVAC General",
```

Comment:
@weilixu Both sections 10 and  19 have the same section name. Should we update one of these section names?

Reply:
Sounds great! I'll move the rules when they are reorganized.